test/rule-matches/inserted-into-focus-order-matches.js

```diff
@@ -0,0 +1,45 @@
+describe('inserted-into-focus-order-matches', function() {
```

Comment:
I think this should also test for non-numerical tabindex values, like tabindex="abc"